core/src/main/resources/lib/form/radioBlock.jelly

```diff
@@ -58,7 +58,7 @@ THE SOFTWARE.
 	  <td colspan="3">
       <label>
 	      <input type="radio" name="${name}" value="${value}"
-             class="radio-block-control block-control" checked="${checked?'true':null}" />
+             class="radio-block-control block-control" checked="${checked?'true':null}" disabled="${displayOnlyMode?'true':null}" />
```

Comment:
Any chance you could format this putting attributes in multiple lines, like on other inputs on this PR?

I'm also in favour of adding spacing within ternary operators. But maybe that's too much for this PR.

Reply:
I went for minimal diff where possible, (although my IDE has fought me to a degree in some files)

happy to format this